shawburn/style-editor.css

```diff
@@ -1602,7 +1608,7 @@ pre.wp-block-verse {
 	z-index: 1;
 }
 
-.fse-template-part .main-navigation > div > ul li:hover, .fse-template-part .main-navigation > div > ul li[focus-within] {
+.fse-template-part .main-navigation > div > ul li:hover, .fse-template-part.js-focus-within .main-navigation > div > ul li[focus-within], .js-focus-within .fse-template-part .main-navigation > div > ul li[focus-within] {
```

Comment:
this is coming from the sass compiler, which we must have updated at some point and haven't built the theme since. I say this doesn't hurt to include, but we could do it separately, your call!

Reply:
Won't this actually break things, because the selector is different?

Reply:
looking at it more closely, I'm not so sure anymore. Looks like something is missing according to [the docs](https://www.npmjs.com/package/postcss-focus-within) (we need the polyfill paired with the CSS change). postcss-focus-within was last updated in https://github.com/Automattic/themes/pull/7059

Reply:
Support for `focus-within` looks pretty good: https://caniuse.com/?search=focus-within. Maybe we should just remove the polyfill?

Reply:
Fix here: https://github.com/Automattic/themes/pull/8570

Reply:
Will this need updating once https://github.com/Automattic/themes/pull/8570 is merged (I'm guessing `.js-focus-within` won't be needed with that PR)?

Reply:
yes